content/webapp/pages/season.tsx

```diff
@@ -55,28 +55,26 @@ const SeasonPage = ({
         Body={<Body body={season.body} pageId={season.id} />}
       />
 
-      {events.length > 0 && (
+      {(exhibitions.length > 0 || events.length > 0) && (
         <SpacingSection>
```

Comment:
You could create a value called `eventsAndExhibitions = [...events, ...exhibitions]` and just do a check on that.

Assuming it's correct that we want events first? Just because we do it differently elsewhere.